src/LogDensityProblems.jl

```diff
@@ -12,14 +12,13 @@ documentation.
 """
 module LogDensityProblems
 
-export TransformedLogDensity, ADgradient
+export ADgradient
 
 using ArgCheck: @argcheck
 using DocStringExtensions: SIGNATURES, TYPEDEF
-using UnPack: @unpack
-import Random
+using Random: AbstractRNG, GLOBAL_RNG
```

Comment:
FYI IIRC for Julia >= 1.3 `Random.default_rng` (https://docs.julialang.org/en/v1.9-dev/stdlib/Random/#Random.default_rng) is the officially documented way for getting the default RNG that is e.g. also used by `seed!(::Int)` etc. There were also some discussions on Slack, and it seems `default_rng` is preferred when it exists as `GLOBAL_RNG` is available as it is only kept in a slightly weird way for backwards compatibility (https://github.com/JuliaLang/julia/blob/6008cbc019424a53786552a511bbef92099c033a/stdlib/Random/src/RNGs.jl#L326-L328).